### DISCUSSION TITLE
We are non profit organization

### DISCUSSION THREAD

**Discussion by lightoflifetrust** [community/community#126518] [GitHub Education]
### Hi everyone,

We are non profit organization based on India .
We are running more then 20 Schools in Maharashtra.
We are also provide free education to those child who cant afford school fees,  who are belong to poor family.
We are also provide to 
sponsorship to those child who want to continue there higher education.
Few month ago we have heard GitHub running discount program for school, institution, college.
we have applied for GitHub campus program.
If our application is approved by GitHub team its really helpful for our children's to continue there traditional education with digital education.

**Comment by aliffian425** [comment]
bless you for the incredible work you're doing, and hopefully everything goes as u planned

Additionally GitHub does offer discounted plans for non-profit organizations through their Non-Profit Organization program. You can find more information about this program and the application process on the GitHub website: [Here](https://github.com/nonprofit)

**Comment by lightoflifetrust** [comment]
We have already get GitHub Non Profit Discount, 
But We need GitHub Discount for Our Schools.
I hope GitHub team will approved soon our application.
